apis/database.graphql

```diff
@@ -570,12 +566,35 @@ input DataPropositionInput {
   """
   and: [DataPropositionInput!]
 
+  """
+  True for data for which the proposition `not` is false, otherwise false.
+  """
+  not: DataPropositionInput
+
+  """
+  True for data for which at least one proposition in `or` is true, otherwise
+  false. In particular, it is false if there is no proposition in `or`.
+  """
+  or: [DataPropositionInput!]
+
+  """
+  True for data for the component with identifier `formatId`, otherwise
+  false.
+  """
+  formatId: UuidPropositionInput
```

Comment:
Please use `dataFormatId` as field name (because it is not clear from the context that the format refers to the format of data). I would also change the description to:

> True for data with at least one resource for whose format identifier the proposition `dataFormatId` is true, otherwise false. See `GetHttpsResource#formatId` and `FileMetaInformation#formatId`.

Reply:
In 263e8c3fa8a3f645cdb62a0344078da7f0969e0b , I have substituted all `formatId` by `dataFormatId`. Or should we continue to use `formatId` for `GetHttpsResource` and `FileMetaInformation`?

Reply:
Ah, yes, renaming all fields seems like a good idea. The latter two are breaking changes though and need to be communicated with Stephen so that he adapts his implementation.

Reply:
@StephenCzarnecki We plan a breaking change to rename `formatId` to `dataFormatId`. Is this ok for you?

Reply:
@christoph-maurer Yes, that is fine.  It is an easy thing to change.